public/CHANGELOG.md

```diff
@@ -0,0 +1,58 @@
+# OSIM Changelog
```

Comment:
Does this mean we now need to keep updating both `CHANGELOG.md` and `public/CHANGELOG.md`? If so, is there way how to either keep just this one or somehow provide a "link" to the one in a root? I mean, having two CHANGELOG files calls for inconsistencies.

Reply:
This is only in public for local development purposes--I change the content of the file to reflect that

Reply:
Actually a symlink might be better here...